java/src/jmri/util/swing/JmriBeanComboBox.java

```diff
@@ -63,7 +63,7 @@ public JmriBeanComboBox(Manager inManager, NamedBean inNamedBean, DisplayOptions
         //fires when drop down list item is selected
         addItemListener((ItemEvent event) -> {
             if (event.getStateChange() == ItemEvent.SELECTED) {
-                JmriBeanComboBox cb = (JmriBeanComboBox) event.getSource();
+                event.getSource();
```

Comment:
Again, this statement does absolutely nothing. Did you evaluate that each change you made to remove unused code did not leave useless code in place?